src/post.proj

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Import Project="dir.props" />
+  <Import Project="..\dir.targets" />
```

Comment:
This file provides some default targets like BuildAndTest which if is missing will cause the build to fail.  Seems like the pattern is to include this file to stub out some default expected targets.